source/strings.tex

```diff
@@ -4879,6 +4895,85 @@
 \end{codeblock}
 \end{itemdescr}
 
+\indexlibrarymember{operator+}{basic_string}%
+\begin{itemdecl}
+template<class charT, class traits, class Allocator>
+  constexpr basic_string<charT, traits, Allocator>
+    operator+(const basic_string<charT, traits, Allocator>& lhs,
+              type_identity_t<basic_string_view<charT, traits>> rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Equivalent to:
+\begin{codeblock}
+basic_string<charT, traits, Allocator> r = lhs;
+r.append(rhs);
+return r;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{basic_string}%
+\begin{itemdecl}
+template<class charT, class traits, class Allocator>
+  constexpr basic_string<charT, traits, Allocator>
+    operator+(basic_string<charT, traits, Allocator>&& lhs,
+              type_identity_t<basic_string_view<charT, traits>> rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Equivalent to:
+\begin{codeblock}
+lhs.append(rhs);
+return std::move(lhs);
```

Comment:
Should we drop `std::move()` here as `lhs` is move-eligible? (ditto below)

Reply:
@jwakely , any opinion here?

Reply:
Since this makes no difference here, let's stick with what was approved for the paper.

Reply:
Yeah I don't mind either way.